ocs_sample_library_preview/BaseClient.py

```diff
@@ -23,6 +23,7 @@ def __init__(self, api_version: str, tenant: str, url: str, client_id: str = Non
             self.__auth_object = None
 
         self.__uri_api = url + '/api/' + api_version
+        self.__session = requests.Session()
```

Comment:
does this say this is deprecated? https://2.python-requests.org/projects/3/_modules/requests/sessions/ (very bottom)

Reply:
wait thats a different thing, all good